src/main/java/de/retest/recheck/printer/TestReportPrinter.java

```diff
@@ -0,0 +1,23 @@
+package de.retest.recheck.printer;
+
+import java.util.stream.Collectors;
+
+import de.retest.recheck.ignore.ShouldIgnore;
+import de.retest.recheck.report.TestReport;
+
+public class TestReportPrinter implements Printer<TestReport> {
```

Comment:
This class is used in?

Reply:
Mostly recheck.cli to create a detailed version of `toString()` (formerly via `toDetailedString()`). We have decoupled these concerns.